easy/17.js

```diff
@@ -1,10 +1,10 @@
 function firstVowel(str) {
-	
+  return str.indexOf(str.match(/[aeiou]/gi)[0]);

```

Comment:
Uncaught TypeError: Cannot read property '0' of null - error if there is nl vowel. You cannot use index when you are not sure that array/string has any value. Better remove flag global (g) and index. Then it should return first appearance of vowel